src/main/java/com/zenjava/javafx/maven/plugin/AbstractJfxToolsMojo.java

```diff
@@ -73,7 +73,7 @@
      * <p>
      * This defaults to 'target/jfx/app' and in most cases there is no real need to change this.
      *
-     * @parameter property="jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
+     * @parameter property="jfx.jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
```

Comment:
If you are using the maven-invoker-plugin, you should get "informed" about changes very fast, but there is no intention for changing them more than they are now.